### DISCUSSION TITLE
.....

### DISCUSSION THREAD

**Discussion by maurpz** [Launch-X-Latam/MisionFrontEnd#127] [General]
..

**Comment by JosafatJimenezB** [comment]
estaria chido lo de clonar paginas, me uno!

**Reply by JosafatJimenezB** [reply]
enviame un mail por mi perfil, recuerda lo de los grupos

**Reply by Amhedriel** [reply]
Buenas chi@s, les recuerdo que esta prohibido crear grupos en redes sociales fuera de GitHub, revisen los videos que ahí lo dicen unas cuantas veces.
